android/java/org/chromium/chrome/browser/BraveRewardsOnboardingPagerAdapter.java

```diff
@@ -46,18 +55,80 @@
+            && position == (WITH_8_OPTIONS-2)) {
+            view = LayoutInflater.from(ContextUtils.getApplicationContext()).inflate(R.layout.brave_rewards_onboarding_ac_layout, null);
+            RadioGroup hourRadioGroup = view.findViewById(R.id.hour_radio_group);
+            int adsPerHour = BraveRewardsNativeWorker.getInstance().GetAdsPerHour();
+            if (adsPerHour == 1) {
+                        ((RadioButton)view.findViewById(R.id.hour_1_radio)).setChecked(true);
+                    } else if (adsPerHour == 2) {
+                        ((RadioButton)view.findViewById(R.id.hour_2_radio)).setChecked(true);
+                    } else if (adsPerHour == 3) {
+                        ((RadioButton)view.findViewById(R.id.hour_3_radio)).setChecked(true);
+                    } else if (adsPerHour == 4) {
+                        ((RadioButton)view.findViewById(R.id.hour_4_radio)).setChecked(true);
+                    } else if (adsPerHour == 5) {
+                        ((RadioButton)view.findViewById(R.id.hour_5_radio)).setChecked(true);
+                    }
+            hourRadioGroup.setOnCheckedChangeListener(new RadioGroup.OnCheckedChangeListener() {
+                @Override
+                public void onCheckedChanged(RadioGroup radioGroup, int checkedId) {
+                    int hour = adsPerHour;
+                    if (checkedId == R.id.hour_1_radio) {
+                        hour = 1;
+                    } else if (checkedId == R.id.hour_2_radio) {
+                        hour = 2;
+                    } else if (checkedId == R.id.hour_3_radio) {
+                        hour = 3;
+                    } else if (checkedId == R.id.hour_4_radio) {
+                        hour = 4;
+                    } else if (checkedId == R.id.hour_5_radio) {
+                        hour = 5;
+                    }
+                    BraveRewardsNativeWorker.getInstance().SetAdsPerHour(hour);
+                }
+            });
+
+            RadioGroup contributeRadioGroup = view.findViewById(R.id.contribute_radio_group);
+            BraveRewardsNativeWorker.getInstance().SetAutoContributionAmount(15);
+            ((RadioButton)view.findViewById(R.id.contribute_15_radio)).setChecked(true);
+            contributeRadioGroup.setOnCheckedChangeListener(new RadioGroup.OnCheckedChangeListener() {
+                @Override
+                public void onCheckedChanged(RadioGroup radioGroup, int checkedId) {
+                    double contribute = 15;
+                    if (checkedId == R.id.contribute_5_radio) {
+                        contribute = 5;
+                    } else if (checkedId == R.id.contribute_15_radio) {
+                        contribute = 15;
+                    } else if (checkedId == R.id.contribute_25_radio) {
+                        contribute = 25;
+                    } else if (checkedId == R.id.contribute_50_radio) {
+                        contribute = 50;
+                    }
```

Comment:
Feels like a `switch` statement might be a better fit here than an if-else, but it's up to you.

Reply:
@emerick for switch-case, `R.id.contribute_50_radio` should be constant so we can't use it here.

Reply:
Ah OK, I didn't look carefully enough sorry!